tasks/terraform-env.yml

```diff
@@ -28,8 +28,10 @@ run:
       cd census-rm-terraform
       tfenv install
 
-      DB_PASSWORD=$(kubectl get secret db-credentials -o=jsonpath="{.data.password}" | base64 --decode)
-      SFTP_USER_NAME=$(kubectl get secret sftp-ssh-credentials -o=jsonpath="{.data.username}" | base64 --decode)
-      SFTP_PASSPHRASE=$(kubectl get secret sftp-ssh-credentials -o=jsonpath="{.data.passphrase}" | base64 --decode)
-
-      SKIP_REPLICA_DB_SSL_RESET=true SKIP_DB_SSL_RESET=true AUTO_APPLY=true ./apply-prod.sh "db_user_password=$DB_PASSWORD" "sftp_user_name=$SFTP_USER_NAME" "sftp_user_password=$SFTP_PASSPHRASE"
+      set +x
+      echo "db_user_password = \"$(kubectl get secret db-credentials -o=jsonpath='{.data.password}' | base64 --decode)\"' >> secret.tfvars
+      echo "sftp_user_name = \"$(kubectl get secret sftp-ssh-credentials -o=jsonpath='{.data.username}' | base64 --decode)"' >>  secret.tfvars
+      echo "sftp_user_password = \"$(kubectl get secret sftp-ssh-credentials -o=jsonpath='{.data.passphrase}' | base64 --decode)"' >> secret.tfvars
```

Comment:
You _might_ need to remove whitespace around the `=` as I'm not sure if it would parse correctly?

Also, you probably wouldn't need to add in the escaped double-quotes in your command substitution as I think any spaces would be encoded and would be fine? e.g. 
`echo "db_user_password=$(kubectl get secret db-credentials -o=jsonpath='{.data.password}' | base64 --decode)" >> secret.tfvars`

Always tricky to tell until the thing actually runs though!

Reply:
The format of our existing `tfvars` files are all `key = "value"` so I think this should be okay as is.

Reply:
Actually the commands were missing and mismatching quotes though, fixed now